src/main/resources/db/migration/V1_116__referral_start_date.sql

```diff
@@ -0,0 +1,4 @@
+alter table dynamic_framework_contract
+    add column referral_start_date date not null default '2021-01-01';
```

Comment:
_zero-downtime analysis_

For PostgreSQL 10, this will lock the table until all values are propagated.

> Introducing a default value at the same time of adding the column will cause the table to be locked while the default value in propagated for all rows in the table

> In the recently release PostgreSQL 11, this is no longer the case for non-volatile default values

source: https://medium.com/paypal-tech/postgresql-at-scale-database-schema-changes-without-downtime-20d3749ed680#bddc

This is still **fine** as we have about ~120 contracts in total, just wanted to flag this for future changes.